components/announcement_bar/announcement_bar.jsx

```diff
@@ -82,20 +92,20 @@ export default class AnnouncementBar extends React.PureComponent {
             return {message: error.message, color: null, textColor: null, type: error.type, allowDismissal: true};
         }
 
-        const bannerText = global.window.mm_config.BannerText || '';
-        const allowDismissal = global.window.mm_config.AllowBannerDismissal === 'true';
-        const bannerDismissed = localStorage.getItem(StoragePrefixes.ANNOUNCEMENT + global.window.mm_config.BannerText);
+        const bannerText = this.props.bannerText || '';
+        const allowDismissal = this.props.allowBannerDismissal;
```

Comment:
A follow on change to this is to pass the parameter to the config/license endpoints that returns booleans instead of string representations of same, allowing us to remove all the `=== 'true'` comparisons upstream as well :)